dockers/base-cuda/Dockerfile

```diff
@@ -114,7 +114,7 @@ RUN \
     rm -rf apex
 
 RUN \
-    # install DeepSpeed
+    # install DeepSpeed and FairScale
```

Comment:
Eventually we should add FairScale to the CI, but I think this can be done in a separate PR